### PR TITLE
Updates to fix issue

### LINKED ISSUE #15414: Test data and predictions possible error
When evaluating the accuracy of the model, aren't the predictions supposed to be created against the test data, i.e. model.Transform(testData) instead of model.Transform(trainingData)?
And when doing so, the sample program throws an exception because of an issue in the test-tags.tsv file. The first test file name there  is broccoli2.png but it should be broccoli2.jpg.

[AB#1608693](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1608693)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7ca0a402-fa67-fbd4-4cbc-604b4b59c97c
* Version Independent ID: c8a996eb-82ed-a4bd-94d8-ef0b32158c67
* Content: [Tutorial: Generate an ML.NET image classification model from a pre-trained TensorFlow model - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification#feedback)
* Content Source: [docs/machine-learning/tutorials/image-classification.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/image-classification.md)
* Product: **dotnet-ml**
* GitHub Login: @natke
* Microsoft Alias: **nakersha**

### PR DESCRIPTION
**Pull request by luisquintanilla**
Updates to fix issue

Code Sample PR: dotnet/samples#1680

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/image-classification?branch=pr-en-us-15498)

Fixes #15414

### CODE REVIEW

**Review by cjgronlund — APPROVED** [review]
Looks good!
